### PR TITLE
only download pytorch model

### LINKED ISSUE #20: model-loader-huggingface: Prevent downloading both safe tensors and pytorch model
It's a waste of storage space to download both the safe tensors model and the pytorch pickle model.

For example llama2 model contains both: https://huggingface.co/meta-llama/Llama-2-7b-hf/tree/main

### PR DESCRIPTION
**Pull request by samos123**
Fixes #20

This reduces the model size of many models by 2x or 3x because safetensors and tensorflow model doesn't get used by any of our images. That being said, we might want to consider exposing this as a param in a follow up PR.

### CODE REVIEW

File: model-loader-huggingface/Dockerfile [hunk 1]
```diff
@@ -1,11 +1,12 @@
 ARG BASE_IMAGE=substratusai/base:latest
 FROM ${BASE_IMAGE}
 
+RUN mkdir -p /content/model
 
 COPY requirements.txt requirements.txt
 RUN pip3 install --no-cache-dir -r requirements.txt
 
-COPY scripts/* scripts
+COPY scripts/* scripts/
```
Comment (brandonjbjelland) [inline model-loader-huggingface/Dockerfile:9]: good catch 🎣

Comment (samos123) [reply model-loader-huggingface/Dockerfile:9]: it was caught by kaniko that threw up on it

File: model-loader-huggingface/src/test_utils.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    assert filter_files(files) == files[:1]
+
+    files = ["config.json",
+             "model-00001-of-00002.safetensors",
+             "model-00002-of-00002.safetensors",
+             "model.safetensors.index.json",
+             "pytorch_model-00001-of-00002.bin",
+             "pytorch_model-00002-of-00002.bin",
+             "pytorch_model.bin.index.json"]
+    assert filter_files(files) == [files[0], files[3], files[4], files[5], files[6]]
```
Comment (brandonjbjelland) [inline model-loader-huggingface/src/test_utils.py:17]: Same tests just in a different form. `pytest.parametrize` is good for table-based testing:

```suggestion
@pytest.mark.parametrize(
    "files, expected",
    [
        (
            [
                "pytorch_model.bin",
                "tf_model.h5",
            ],
            ["pytorch_model.bin"],
        ),
        (
            [
                "config.json",
                "model-00001-of-00002.safetensors",
                "model-00002-of-00002.safetensors",
                "model.safetensors.index.json",
                "pytorch_model-00001-of-00002.bin",
                "pytorch_model-00002-of-00002.bin",
                "pytorch_model.bin.index.json",
            ],
            [
                "config.json",
                "model.safetensors.index.json",
                "pytorch_model-00001-of-00002.bin",
                "pytorch_model-00002-of-00002.bin",
                "pytorch_model.bin.index.json",
            ],
        ),
    ],
)
def test_filter_files(files, expected):
    assert filter_files(files) == expected
```

**Review by brandonjbjelland — APPROVED** [review]
Optional suggestion but LGTM 🚢

**Comment by brandonjbjelland** [comment]
Nice find eliminating the tensorflow cruft here. 👯

File: model-loader-huggingface/src/test_utils.py (hunk 2 above)
Comment (samos123) [reply model-loader-huggingface/src/test_utils.py:17]: I will adopt your test, looks cleaner and clearer for someone reading the test. Thanks for the suggestion!

**Pull request merged**